modules/po/webadmin.de_DE.po

```diff
@@ -200,13 +200,13 @@ msgstr ""
 "UNSICHER!"
 
 #: modules/po/../data/webadmin/tmpl/add_edit_network.tmpl:83
-msgid "Trust the PKI:"
+msgid "Automatically detect trusted certificates (Trust the PKI):"
 msgstr "Vertraue dem PKI:"
```

Comment:
Shouldn't this be somehow marked as "not really translated", or something like that? If I understand this correctly, this is still considered a proper translation.

Reply:
Yes, it depends on the i18n tool. Some of them automatically unverify translations on upload, some require you to clear the string, some of them require manual work